docs/_docs/windows.md

```diff
@@ -193,16 +174,27 @@ RubyInstaller is a self-contained Windows-based installer that includes the Ruby
 
 Optionally you can use [Autoinstall Jekyll for Windows](https://github.com/KeJunMao/fastjekyll#autoinstall-jekyll-for-windows).
 
-#### Auto Regeneration (RubyInstaller method)
 
-Although Jekyll would suggest:
+## Time-Zone Management 
+Since Windows doesn't have a native source of zoneinfo data, the Ruby Interpreter would not understand IANA Timezones and hence using them had the `TZ` environment variable default to UTC/GMT 00:00.
+Though Windows users could alternatively define their blog's timezone by setting the key to use POSIX format of defining timezones, it wasn't as user-friendly when it came to having the clock altered to changing DST-rules.
 
-```
-Please add the following to your Gemfile to avoid polling for changes:
-    gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+Jekyll now uses a rubygem to internally configure Timezone based on established [IANA Timezone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+While 'new' blogs created with Jekyll v3.4 and greater, will have the following added to their 'Gemfile' by default, existing sites *will* have to update their 'Gemfile' (and installed) to enable development on Windows:
+
+```ruby
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 ```
 
-Auto regeneration will work fine without including `gem 'wdm'`.
+## Auto Regeneration 
+As of v1.3.0, Jekyll uses the `listen` gem to watch for changes when the
+`--watch` switch is specified during a build or serve. While `listen` has
+built-in support for UNIX systems, it requires an extra gem for compatibility
+with Windows. Add the following to the Gemfile for your site:
 
-#### Time-Zone Management (RubyInstaller method)
-The [time-zone](/docs/windows/#timezone-management) documentation applies for this method.
+```ruby
+gem 'wdm', '~> 0.1.0' if Gem.win_platform?
+```
+
+```
```

Comment:
@pathawks it's odd because when I click edit on my end, line 200 doesn't exist. I made another commit, hopefully it's gone for real this time.